src/cursor/cur_hs.c

```diff
@@ -145,9 +121,35 @@ __wt_hs_cursor_search_near(WT_SESSION_IMPL *session, WT_CURSOR *cursor, int *exa
     return (ret);
 }
 
+/*
+ * __curhs_copy_key --
+ *     Copy the key from file cursor to the history store cursor.
+ */
+static inline void
+__curhs_copy_key(WT_CURSOR *hs_cursor, WT_CURSOR *file_cursor)
```

Comment:
I guess this is subjective, but I think the name is misleading. In my mind, copy usually means deep copy which makes it sound like it's memcpying the key buffer somewhere.

Reply:
Only improvement I can think of is `__curhs_copy_key_ptr'.

Reply:
Fixed in 1e1d5ed.